OfficeApplication/OfficeApplication/OfficeUtil.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace OfficeApplication
+{
+    class OfficeUtil
+    {
+        public  Employee GetEmployeeDetails()
+        {
+            int id, deptId;
+            decimal salary;
+            string name,address;
+            Console.Write("Enter Employee Id: ");
+            id = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Enter Employee Name: ");
+            name = Console.ReadLine();
+            Console.Write("Enter Employee Address: ");
+            address = Console.ReadLine();
+            Console.Write("Enter Employee Salary: ");
+            salary = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Select Department Id: ");
+            Console.WriteLine("1. Administration");
```

Comment:
Show department list fetched from db not hardcoted list use different function to display department